chart/templates/argo-rollouts.yaml

```
@@ -9,7 +9,7 @@ metadata:
 spec:
   project: default
   source:
-    repoURL: https://github.com/aws-samples/eks-blueprints-add-ons
+    repoURL: {{ .Values.repo_url }}
```

I think we want to use 
```suggestion
    repoURL: {{ .Values.repoUrl }}
```

For these since https://github.com/aws-samples/eks-blueprints-add-ons/blob/67452e2f7f2cea2488e2da5b6ba981c549f4635c/chart/values.yaml#L2 - correct?